loopdb/sqlc/queries/static_address_loopin.sql

```diff
@@ -0,0 +1,79 @@
+-- name: InsertStaticAddressLoopIn :exec
+INSERT INTO static_address_swaps (
+    swap_hash,
+    swap_invoice,
+    last_hop,
+    quoted_swap_fee,
+    deposit_outpoints,
+    htlc_tx,
+    htlc_tx_fee_rate,
+    htlc_timeout_sweep_tx,
+    htlc_timeout_sweep_address
+) VALUES (
+    $1,
+    $2,
+    $3,
+    $4,
+    $5,
+    $6,
+    $7,
+    $8,
+    $9
+);
+
+-- name: UpdateStaticAddressLoopIn :exec
+UPDATE static_address_swaps
+SET
+    htlc_tx = $2,
+    htlc_tx_fee_rate = $3,
+    htlc_timeout_sweep_tx = $4,
+    htlc_timeout_sweep_address = $5
```

Comment:
Why does htlc_timeout_sweep_address need to be updated?

Reply:
Thanks for catching this. It does not need an update because it is created in the db in action `InitHtlcAction` and won't change for the rest of the swap flow.